src/test/java/SpeechRankGUITests/src/main/java/page/objects/Confitura2018.java

```diff
@@ -0,0 +1,82 @@
+package page.objects;
+
+import driver.manager.DriverManager;
+import org.apache.logging.log4j.LogManager;
+import org.openqa.selenium.interactions.Actions;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import waits.WaitForElement;
+import org.openqa.selenium.JavascriptExecutor;
+
+
+public class Confitura2018 {
```

Comment:
does it really only represent Confitura2018 conference or any conference? Maybe `ConferencePage` or `ConferencePageObject` would be a better name?